csharp/src/lesson01/exercise/Hello.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using OpenTracing.Tutorial.Library;
+
+namespace OpenTracing.Tutorial.Lesson01.Exercise
+{
+    internal class Hello
```

Comment:
might be useful to start with the version that uses Logging, but does not use any tracing code.

Reply:
Sound fine to me